app/views/admins/_user_entry.mobile.haml

```diff
@@ -0,0 +1,70 @@
+
+%li.user.media
+  %div.pull-left
+    - if user.person
+      %span.media-object
+        = person_image_tag(user.person, size: :thumb_small)
+
+  %div.media-body.row
+    %div.pull-right
+      %span.label.label-default
+        = t('.id')
+        = user.id
+      %span.label.label-info
+        = t('.guid')
+        = user.person.guid if user.person
+
+    %h4.media-heading
+      = user.person.name if user.person
+
+    %div
+      .unstyled.text-right.actions
+        = link_to t('admins.user_search.view_profile'), person_path(user.person), class: 'btn btn-default btn-block'
+        = link_to t('admins.user_search.add_invites'), add_invites_path(user.invitation_code), class: 'btn btn-info btn-block'
+        - unless user.person.closed_account
+          = link_to t('admins.user_search.close_account'), admin_close_account_path(user), method: :post, data: { confirm: t('admins.user_search.are_you_sure') }, class: 'btn btn-danger btn-block'
+
+        - unless user.closed_account?
+          - unless user.access_locked?
+            = link_to t('admins.user_search.lock_account'), admin_lock_account_path(user), method: :post, data: { confirm: t('admins.user_search.are_you_sure_lock_account') }, class: 'btn btn-danger btn-block'
+          - else
+            = link_to t('admins.user_search.unlock_account'), admin_unlock_account_path(user), method: :post, data: { confirm: t('admins.user_search.are_you_sure_unlock_account') }, class: 'btn btn-danger btn-block'
```

Comment:
- Line is too long. [215/120]
- Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
- Space inside { detected.
- Space inside } detected.